app/views/dashboard/_stripe_account_setup_banner.njk

```diff
@@ -1,4 +1,5 @@
 {% if permissions.stripe_account_details_update %}
```

Comment:
I think we need an `{% if showFeatureStripeOnboarding %}` now. Before we were checking that in the controller.